### PR TITLE
Update myhome homepage not home page

### PR DESCRIPTION
**Pull request by jordesign**
#### Changes proposed in this Pull Request

Addresses #53036 by updating checklist references in My Home to be 'Homepage' instead of Home Page

#### Testing instructions

Start a new site (so checklist is fresh/unchecked). Check the 'Edit Homepage' option with /home/.

(Note - I'm not sure what state/context is needed to see the 'Choose a theme' checklist item - but I have updated the tet in there too).

Related to #53036

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -239,9 +239,10 @@
 		"autoprefixer": "^10.2.5",
 		"component-event": "^0.2.0",
 		"component-query": "^0.0.3",
-		"react-test-renderer": "^16.12.0",
+		"eslint-plugin-you-dont-need-lodash-underscore": "^6.12.0",
 		"pkg-dir": "^5.0.0",
 		"postcss-custom-properties": "^11.0.0",
+		"react-test-renderer": "^16.12.0",
```
Comment (p-jackson) [inline client/package.json]: This is probably just a hangover from some earlier work you were doing?

```suggestion
		"react-test-renderer": "^16.12.0",
		"pkg-dir": "^5.0.0",
		"postcss-custom-properties": "^11.0.0",
```

**Review by p-jackson — COMMENTED** [review]
LGTM 👍 
Screenshot 2021-05-24 at 2 06 39 PM

The change to `package.json` looks unrelated to me, we should tidy that up.

I'm not sure what the problem is with the e2e tests 🤔

**Comment by p-jackson** [comment]
Oh, it looks like `Unit tests (Web app)` won't pass until the `package.json` change is removed. It's causing a `yarn.lock` file change which is an error.

**Comment by jordesign** [comment]
Thanks @p-jackson - I reverted that original change and have done so again without the package.json change. Fingers crossed.

**Review by p-jackson — APPROVED** [review]

**Pull request merged**